src/Sema.zig

```diff
@@ -8720,25 +8720,29 @@ fn zirEmbedFile(sema: *Sema, block: *Block, inst: Zir.Inst.Index) CompileError!A
             return sema.fail(block, operand_src, "unable to open '{s}': {s}", .{ name, @errorName(err) });
         },
     };
+    if (result.embed_file) |embed_file| {
```

Comment:
Style nit but I prefer using `orelese` in cases like this to avoid extra indentation:
```zig
const embed_file = result.embed_file orelse {
    defer mod.gpa.free(result.real_path);
    return sema.fail(block, operand_src, "unable to load '{s}': canonical file system name is '{s}'", .{ name, result.real_path });
};
```